pilot/pkg/features/pilot.go

```diff
@@ -216,8 +216,15 @@ var (
 		true,
 		"If enabled, for a headless service/stateful set in Kubernetes, pilot will generate an "+
 			"outbound listener for each pod in a headless service. This feature should be disabled "+
-			"if headless services have a large number of pods. ",
+			"if headless services have a large number of pods.",
 	)
+
+	BlockHTTPonHTTPSPort = env.RegisterBoolVar(
+		"PILOT_BLOCK_HTTP_ON_HTTPS_PORT",
```

Comment:
on_443 ?

Reply:
we could also do block https on 80

Reply:
I don't think https on 80 breaks stuff though since it won't sni match? I'll try it

Reply:
Yeah it doesn't seem to be an issue